htdocs/admin/facture.php

```diff
@@ -51,6 +51,32 @@
 
 include DOL_DOCUMENT_ROOT.'/core/actions_setmoduleoptions.inc.php';
 
+// Action mise a jour ou ajout d'une constante
```

Comment:
Can you confirm this code is not duplicate with the code into:
include DOL_DOCUMENT_ROOT.'/core/actions_setmoduleoptions.inc.php';